wp-content/themes/workingnyc/src/js/modules/search-archive.js

```diff
@@ -78,17 +78,112 @@ export default {
 
       /**
        * This is the endpoint list for terms and post requests
+       * The other archive files have an endpoint for filter terms, but the search page sets filter terms locally
        *
        * @type  {Object}
        *
-       * @param  {String}  terms  A required endpoint for the list of filters
-       * @param  {String}  jobs   This is based on the 'type' setting above
+       * @param  {String}  search   The API endpoint that returns search results
        */
       endpoints: {
-        terms: '/wp-json/api/v1/terms/?post_type[]=jobs&cache=0',
         search: '/wp-json/api/v1/searchRelevanssi'
       },
 
+      terms: [ {
+          name: 'Result type',
+          slug: 'result-type',
+          filters: [ {
+              id: 1,
+              name: 'Jobs',
+              slug: 'jobs',
+              parent: 'result-type',
+              checked: false
+            },
+            {
+              id: 2,
+              name: 'Programs',
+              slug: 'programs',
+              parent: 'result-type',
+              checked: false
+            }
+          ]
+        }, 
+        {
```

Comment:
These are the possible values for locations from the terms endpoints for jobs and programs - we may use only some of these for the actual search results page